lib/web_ui/dev/integration_tests_manager.dart

```diff
@@ -175,9 +172,9 @@ class IntegrationTestsManager {
 
     final int numberOfTestsRun = _numberOfPassedTests + _numberOfFailedTests;
 
-    print('INFO: ${numberOfTestsRun} tests run. ${numberOfPassedTests} passed '
-        'and ${numberOfFailedTests} failed.');
-    return numberOfFailedTests == 0;
+    print('INFO: ${numberOfTestsRun} tests run. ${_numberOfPassedTests} passed '
+        'and ${_numberOfFailedTests} failed.');
+    return _numberOfFailedTests == 0;
```

Comment:
This LGTM as an emergency fix, but we also need a test that fails intentionally to verify that when we need to fail we do. We can borrow ideas from other tools that have failure tests:

- [felt screenshot unit tests](https://github.com/flutter/engine/blob/master/lib/web_ui/test/golden_tests/golden_failure_smoke_test.dart)
- [flutter test](https://github.com/flutter/flutter/blob/master/dev/automated_tests/test_smoke_test/fail_test.dart)
- [flutter drive](https://github.com/flutter/flutter/blob/master/packages/flutter_driver/test_driver/failure_test.dart)

Reply:
Great, thanks! I was also thinking about it yesterday. I think that's a very important test case to have.